deps/src/singular.cpp

```diff
@@ -5,6 +5,25 @@
 #include "ideals.h"
 #include "matrices.h"
 
+std::string singular_return;
+std::string singular_error;
+std::string singular_warning;
+extern int  inerror;
+
+void WerrorS_for_julia(const char * s)
```

Comment:
These callbacks should ideally be made `static`, too, as nothing seems to reference those names from outside

Reply:
Indeed.